### PR TITLE
Adds email notification if failure occurs

### PR DESCRIPTION
**Pull request by ives1227**
**Uses the lts-mqemailnotifier to send a message to the queue to trigger the dais-notifier*
* * *

**Ticket Link**: https://jira.huit.harvard.edu/browse/LTSEPADD-93

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)
MQ Email Notifier Module: https://github.com/harvard-lts/mq_email_notifier

# How should this be tested?

Follow testing instructions here: https://github.com/harvard-lts/epadd-curator-app#running-tests

You can also upload a packet with the drsConfig.txt removed to NextCloud Dev (if you have access).  Curator app fails if no drsConfig.txt is found so an email should be sent out.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? yes
- integration tests? yes (unfortunately bundled with unit tests)

### CODE REVIEW

File: app/monitor_exports/__init__.py [hunk 1]
```diff
@@ -128,42 +154,54 @@ def call_dims_ingest(manifest_object_key):
...
+        logger.debug("Putting ingest.txt at prefix: " + manifest_parent_prefix)
         content = "Pending ingest.."
         s3_resource.Object(epadd_bucket_name, os.path.join(manifest_parent_prefix, "ingest.txt")).put(Body=content)
 
     # Remove loading file
     s3_resource.Object(epadd_bucket_name, os.path.join(manifest_parent_prefix, "LOADING")).delete()
 
+def get_notify_emails(payload_data):
+    admin_md = payload_data['admin_metadata']
+    delmiter = ""
```
Comment (awoods) [inline app/monitor_exports/__init__.py:179]: Typo: 'delmiter'. 

Line:185 may hit a `null` error.

**Review by awoods — CHANGES_REQUESTED** [review]

File: app/monitor_exports/__init__.py (hunk 1 above)
Comment (ives1227) [reply app/monitor_exports/__init__.py]: Good catch

File: requirements.txt [hunk 2]
```diff
@@ -2,8 +2,10 @@ requests~=2.27.1
 pytest==6.2.2
 botocore==1.27.55
 boto3==1.24.55
+lts-mqemailnotifier
```
[the remark this answers is not in the document]
Comment (ives1227) [reply requirements.txt]: It is published here right now: https://test.pypi.org/manage/project/lts-mqemailnotifier/release/0.0.2/ - I changed it to specify a number

**Review by awoods — APPROVED** [review]
Looks good.

Separately, can you point me to the documentation you use for publishing Python artifacts. See: https://github.com/harvard-lts/epadd-curator-app/pull/22#discussion_r1167639224

**Comment by ives1227** [comment]
@awoods publishing the artifact is in the Readme of the utility here: https://github.com/harvard-lts/mq_email_notifier#7-build-and-publish-the-package

**Pull request merged**